package.json

```diff
@@ -35,39 +78,47 @@
     "response",
     "csrf",
     "redirect-back",
-    "outlet",
     "client-only",
     "hydrated",
-    "body parser"
+    "server-only",
+    "cors",
+    "rolling cookie",
+    "safe redirect",
+    "typed cookie",
+    "typed session",
+    "client IP address",
+    "client locale",
+    "json hash",
+    "prefetch",
+    "named action"
   ],
   "peerDependencies": {
-    "@remix-run/react": "^1.10.0",
-    "@remix-run/server-runtime": "^1.10.0",
-    "react": "^16.8.0 || ^17.0.0 || ^18.0.0",
+    "@remix-run/react": "^1.19.1",
+    "@remix-run/router": "^1.7.2",
+    "@remix-run/server-runtime": "^1.19.1",
```

Comment:
The hard requirement here for the `^1.x` version of remix means we have to run `npm i` in V2 with the --legacy-peer-deps flag. Would changing this to `>=1.x` solve?